src/ar_config.erl

```diff
@@ -0,0 +1,171 @@
+-module(ar_config).
+-export([parse/1]).
+-include("ar.hrl").
+-include("ar_config.hrl").
+
+%%%===================================================================
+%%% Public interface.
+%%%===================================================================
+
+parse(Config) when is_binary(Config) ->
+	case ar_serialize:json_decode(Config) of
```

Comment:
The jiffy data structure jiffy uses is just legacy. Using lists and maps would make more sense. Now the jiffy stuff is mixed into the `parse_options` functions. How do you feel about translating the `ar_serialize:json_decode/1` return to maps and lists first?